src/Skills.jsx

```diff
@@ -125,11 +125,10 @@ class Skills extends Component {
 
   componentDidMount() {
     const currentId = (this.props.params) ? this.props.params.id : null;
-    if (!currentId) {
+    if (!currentId)
```

Comment:
Just to clean it up, since they were not required. I usually avoid adding braces for one-liners out of habit.